### DISCUSSION TITLE
how to disable material color palette util?

### DISCUSSION THREAD

**Discussion by rbutera** [end-4/dots-hyprland#768] [Q&A]
I love these dotfiles but I'd really like to use the catppuccin color palette instead of automatically deriving it from my wallpaper.

Anyone have a recommendation for achieving this?

**Comment by end-4** [comment]
- You can modify `~/.cache/ags/scss/_material.scss` to your liking
- Note that picking a wallpaper with Ctrl+Super+T will trigger a color generation and overwrite the above file
  - To *only* change the wallpaper, run `swww img /PATH/TO/NEW_WALL`

**Comment by rbutera** [comment]
thanks @end-4 your dotfiles are the best btw!

just incase someone comes across this thread in the future, the actual path of material_colors is `~/.cache/ags/user/generated/material_colors.scss`

**Reply by end-4** [reply]
oh whoops, i tried to answer on mobile without checking that thoroughly... 
thanks for the correction
